lib/NVS/src/Namespace.h

```diff
@@ -5,6 +5,9 @@
 #include <nvs.h>
 #include <iostream>
 
+#include "nvs_flash.h"
```

Comment:
Please use: `#include  <nvs_flash>`, using the form you have is for header files that are in the Current Working Directory